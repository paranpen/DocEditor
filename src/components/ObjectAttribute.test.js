test('snapshot', () => {
  expect('test!').toMatchSnapshot()
})