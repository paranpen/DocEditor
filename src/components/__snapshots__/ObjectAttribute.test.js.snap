exports[`test snapshot 1`] = `"test!"`;
